idea/test-balance/src/common/transfer-balance-process.ts

```diff
@@ -4,21 +4,48 @@ import { transferService } from '../services/transfer.service';
 import { gearService } from '../gear';
 import { testBalanceLogger } from './test-balace.logger';
 import { producer } from '../rabbitmq/producer';
+import { EventEmitter } from 'node:events';
 
-export async function transferBalanceProcess(payload: any, correlationId: string): Promise<void> {
-  let result;
+export const requests: Array<{ payload: any; correlationId: string }> = [];
```

Comment:
What do you think, maybe we can create a `type` by the name `RequestParams` in rabbitmq directory in the types folder, and use this type here. 

```javascript
interface RequestParams {
  payload: any;
  correlationId: string;
}
```

```javascript
const requests = RequestParams[] = [];
```

Reply:
yep